peer/chaincode_shim.proto

```diff
@@ -73,6 +74,20 @@ message GetStateMetadata {
     string collection = 2;
 }
 
+// GetStateMiltiple is the payload of the ChaincodeMessage.
+// It contains the keys to be retrieved from the ledger.
+// If a collection is specified, the keys will be retrieved
+// from the collection (i.e., the private state).
+message GetStateMiltiple {
+    repeated string keys = 1;
+    string collection = 2;
+}
```

Comment:
Should this be `GetStateMultiple`? Could it use a `repeated GetState` field, or is there a limitation that all the reads need to be from the same collection?